packages/autorest.go/test/autorest/mediatypesgroupwithnormailzedoperationname/zz_constants.go

```diff
@@ -52,41 +52,41 @@ func PossibleContentType1Values() []ContentType1 {
 	}
 }
 
-// ContentType1AutoGenerated - Content type for upload
-type ContentType1AutoGenerated string
+// ContentType2 - Content type for upload
+type ContentType2 string
```

Comment:
I'm curious whether we could see a change like this in a module we've shipped but I suppose if we did, we'd simply add a transform to keep the released name.

Reply:
Fortunately, APIs that specify multiple content types are very rare so I don't expect this to be a problem in practice.  You're correct though that if this were to become a problem we can rename via a transform (which is unfortunate as the new names a better IMO).